src/System.Private.CoreLib/shared/System/Globalization/DateTimeFormatInfoScanner.cs

```diff
@@ -30,7 +30,7 @@ namespace System.Globalization
     using StringList = List<string>;
 #else
     using StringStringDictionary = LowLevelDictionary<string, string>;
-    using StringList = LowLevelList<string>;
+    using StringList = List<string>;
```

Comment:
Only purpose of "using StringList" was to abstract away the difference between List and LowLevelList - now it's just "List" so this could be gotten rid of. Maybe the same with StringStringDictionary...